CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 1.11.0-beta-1
+- Bump bcprov-jdk15to18 to 1.69 (#267)
+- ABI Interaction Support for JAVA SDK (#268)
+- Add keyreg txn encoding tests (#269)
+- Stop empty state proof key from being encoded in txn (#270)
+- Add WaitForConfirmation function (#274)
+- Bug fix for logs on PendingTransactionResponse (#275)
```

Comment:
PR #267 was never merged and PRs #269 and #270 did not target develop, so none of these should be in the changelog

Reply:
Removed those PRs from the changelog and updated the PR description.

Reply:
Regenerated PR list.